docs/guides/1229-always-execute.md

```diff
@@ -0,0 +1,24 @@
+---
+slug: /1229/always-execute
+displayed_sidebar: europa
+---
+
+# Always executing an action
+
+Dagger implemented a way to tell Buildkit not to rely on cache for a specific action.
+
+The `docker.#Run` and `core.#Exec` definitions have an `always` field:
+
+```cue
+// If set to true, the cache will never be triggered for that specific action.
+always: bool | *false
```

Comment:
Is the convention, `type | *defaultValue` or `*defaultValue | type`?

I have seen us use both and I would like us to be consistent.